jsonnet/components/node-exporter.libsonnet

```
@@ -181,6 +181,8 @@ function(params)
                               // https://issues.redhat.com/browse/OCPBUGS-1321
                               '--collector.netclass.ignored-devices=^(veth.*|[a-f0-9]{15}|tun[0-9]*|br[0-9]*|ovn-k8s-mp[0-9]*|br-ex|br-int|br-ext)$',
                               '--path.udev.data=/host/root/run/udev/data',
+                              // Disable btrfs collector as btrfs is not included in RHEL kernels
+                              '--no-collector.btrfs'
```

```suggestion
                              '--no-collector.btrfs',
```
This should fix the `fmt-jsonnet` check.